core/base/src/TProcessID.cxx

```diff
@@ -414,7 +414,7 @@ void TProcessID::RecursiveRemove(TObject *obj)
    UInt_t uid = obj->GetUniqueID() & 0xffffff;
    if (obj == GetObjectWithID(uid)) {
       R__WRITE_LOCKGUARD(ROOT::gCoreMutex);
-      if (fgObjPIDs) {
+      if (fgObjPIDs && ((obj->GetUniqueID()&0xff000000)==0xff000000)) {
```

Comment:
```suggestion
      // Only attempt to remove from the map the items that are already
      // registered (because they are associated with a TProcessID with index 
      // greater than 255.  Attempting to remove an item that is not in the map
      // issues a Warning message.  
      if (fgObjPIDs && ((obj->GetUniqueID()&0xff000000)==0xff000000)) {
```